ssh/tunnel.py

```diff
@@ -16,8 +16,6 @@
 
 from pkgpanda.util import write_string
 
-LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
```

Comment:
You have silently glossed over the core problem :stuck_out_tongue: 

> if we have 5 invokers, ...

We should not have five different scripts that have totally custom ways of doing the exact same things. All of the test_* files will be purged soon (except upgrade_vpc).

Why we do not want to go to that pattern right now also has something to do with the fact that we do not have a consistent module in test_util, rather we have two core functions: provisioning (launch.py, aws.py, azure.py, etc) and API harness for a provisioned DC/OS (dcos_api_session.py, helpers.py, marathon.py). Each of those cases has distinct needs. Once these live in more self-centered environment, the centralized logging would be a good goal